src/natcap/invest/hra.py

```diff
@@ -680,6 +676,10 @@ def execute(args):
                 dependent_task_list=[reprojected_vector_task]
             ))
 
+            # Habitats and stressors are rasterized with ALL_TOUCHED=TRUE
+            if name in habitats_info or name in stressors_info:
+                habitat_stressor_vectors.add(target_simplified_vector)
```

Comment:
It appears that the intent was to apply `ALL_TOUCHED=TRUE` when rasterizing habitats and stressors, but that was not happening because `target_simplified_vector`, not `source_filepath`, is used.